src/app/controllers/schedules_controller.rb

```diff
@@ -55,4 +55,22 @@ def ical
     render :text => cal.to_ical, :content_type => 'text/calendar'
   end
 
+  protected
+
+  def schedule(event)
+    Rails.cache.fetch("#{event.cache_key}/schedule") do
```

Comment:
Does the behavior of `event.cache_key` refresh/change once the event changes too? Is that why we don't need to pass the params `force` into this method too?